### PR TITLE
Ashutosh

### PR DESCRIPTION
**Pull request by a19hu**
- I have added my branch

### CODE REVIEW

**Comment by a19hu** [comment]
@VishalZ123 pls check

**Comment by VishalZ123** [comment]
@a19hu
Please draft a descriptive PR including what you did.

File: src/components/navbar/Company.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
```
Comment (VishalZ123) [inline src/components/navbar/Company.js:1]: This was not the part of the issue.

File: src/components/navbar/Eventhighlight.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
```
Comment (VishalZ123) [inline src/components/navbar/Eventhighlight.js:1]: Please make a seperate folder for these components, instead of putting in navbar folder.

**Review by VishalZ123 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**